csharp_dotnetcore/81.Facebook-Events/README.md

```diff
@@ -0,0 +1,44 @@
+﻿
+
+# Concepts introduced in this sample
+
+This sample shows how to integrate consume Facebook specific payloads, such as PostBacks, quick replies and optin events. 
+Since Bot Framework supports multiple Facebook pages for a single bot, we also show how to know the page to which the message was sent, so developers can have custom behavior per page.
```

Comment:
integrate and consume

Reply:
Fixed the integrate and consume part. optin is listed like this in facebook docs, so I'll leave it as-is for easy text search. Let me know if you have another proposal, happy to reconsider.

Reply:
Also changed PostBacks to postbacks to follow facebook docs here.